### PR TITLE
Added shared services section to Virtual WAN paragraph

### PR DESCRIPTION
**Pull request by lpassig**
If the Virtual WAN approach is chosen, IaaS resources like domain controllers and other additional shared services (DNS) need to be deployed in another VNet, since you can't deploy non-network related resources in the the secured vWAN Hub. The created VNet in the identity Subscription, serves as a additional Identity VNet Hub next to the secured vWAN Hub.

### CODE REVIEW

**Comment by PRMerger16** [comment]
@lpassig : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by victorar — CHANGES_REQUESTED** [review]
Please review comment, as added content is covered already in the management group and subscription organization article.

File: docs/ready/enterprise-scale/network-topology-and-connectivity.md [hunk 1]
```diff
@@ -157,6 +157,8 @@ _Figure 1: Virtual WAN network topology._
 
 - Connect Virtual WAN hubs to on-premises datacenters by using ExpressRoute.
 
+- If the usage of additional shared services like Active Directory domain controllers and DNS servers is required, deploy them in the dedicated [identity subscription](/ready/enterprise-scale/management-group-and-subscription-organization), in an [additional dedicated VNET Hub](azure/architecture/reference-architectures/hybrid-networking/shared-services) next to the Virtual WAN Hub.
```
[the remark this answers is not in the document]
Comment (BrianBlanchard) [reply docs/ready/enterprise-scale/network-topology-and-connectivity.md:160]: @lpassig, @victorar,

Is this PR ready to merge? or are you still working on verbiage.

[the remark this answers is not in the document]
Comment (lpassig) [reply docs/ready/enterprise-scale/network-topology-and-connectivity.md:160]: @victorar In my opinion it's confusing for the reader if shared services are mentioned in the Hub and Spoke paragraph, but not when it comes to the VWAN Paragraph. Could we maybe built mention the reference the pointed out here? We can get rid of the mentioned VNet part if you like.

[the remark this answers is not in the document]
Comment (victorar) [reply docs/ready/enterprise-scale/network-topology-and-connectivity.md:160]: That sounds good and we can keep consistency with hub & spoke. Let's just rephrase the sentence to something like this:

Deploy required shared services (such as DNS servers) in a dedicated landing zone, as such resources cannot be deployed on a Virtual WAN hub.

**Comment by PRMerger14** [comment]
@lpassig : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by victorar — APPROVED** [review]
Thanks! Looks good to me.

**Comment by PRMerger18** [comment]
@lpassig : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by victorar** [comment]
@lpassig  - please remove non-networking related topics from this PR. This PR was opened just to address the note for deploying shared services in the VWAN model. 

If you want to update other articles, please open a separate PR.

**Comment by lpassig** [comment]
Additionally for @BrianBlanchard Added "Carbon footprint reduction" as a motivation in the "Why are we moving to the cloud?" section in docs/strategy/motivations.md

**Comment by PRMerger7** [comment]
@lpassig : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by victorar** [comment]
This PR is not ready for merging. @lpassig , please submit your suggested changes to other articles on a separate PR.

**Comment by PRMerger17** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**